README.md

```diff
@@ -32,7 +32,11 @@ at their own discretion.
     help projects adopt and iterate on policies and processes that achieve
     these goals and eventually allow them to graduate the incubation phase.
 
-      * Mikeal Rogers (@mikeal)
+      * Mikeal Rogers (@mikeal) Currently assigned to Express.
+      * James Snell (@jasnell) Currently assigned to Express.
+      * Rod Vagg (@rvagg) Currently assigned to libuv.
+      * Alexis Campailla (@orangemocha) Currently assigned to libuv.
+
```

Comment:
I am not sure how this works in detail. I know that I got tons of notifications from PRs like https://github.com/marekventur/dependency-updater/pull/21 because my alias is in the readme. 
In any case, it's not a huge issue either way :)